central/graphql/resolvers/plotted_node_vulnerabilities.go

```diff
@@ -49,7 +75,41 @@ func (pvr *PlottedNodeVulnerabilitiesResolver) BasicNodeVulnerabilityCounter(_ c
 
 // NodeVulnerabilities returns the node vulnerabilities for top risky nodes scatter-plot
 func (pvr *PlottedNodeVulnerabilitiesResolver) NodeVulnerabilities(ctx context.Context, args PaginatedQuery) ([]NodeVulnerabilityResolver, error) {
-	vulnResolvers, err := unwrappedPlottedVulnerabilities(ctx, pvr.root, pvr.all, args)
+	if !features.PostgresDatastore.Enabled() {
+		vulnResolvers, err := unwrappedPlottedVulnerabilities(ctx, pvr.root, pvr.all, args)
+		if err != nil {
+			return nil, err
+		}
+
+		ret := make([]NodeVulnerabilityResolver, 0, len(vulnResolvers))
+		for _, resolver := range vulnResolvers {
+			ret = append(ret, resolver)
+		}
+		return ret, nil
+	}
+
+	q, err := args.AsV1QueryOrEmpty()
+	if err != nil {
+		return nil, err
+	}
+
+	if len(pvr.all) == 0 {
+		return nil, nil
+	}
+
+	cvesInterface, err := paginationWrapper{
+		pv: q.GetPagination(),
+	}.paginate(pvr.all, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	vulns, err := pvr.root.NodeCVEDataStore.GetBatch(ctx, cvesInterface.([]string))
+	if err != nil {
+		return nil, err
+	}
+
+	vulnResolvers, err := pvr.root.wrapNodeCVEs(vulns, err)
```

Comment:
Create a raw query for cve ids with pagination and redirect to top-level node vulnerability resolver.